dts/bindings/video/video-interfaces.yaml

```
@@ -0,0 +1,133 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+# Common properties for video interface endpoints.
+
+include: base.yaml
+
+compatible: "zephyr,video-interfaces"
+
+description: |
+  A video pipeline usually consists of several devices, e.g. camera sensors, video
+  data receivers, video data processors, etc. Data interfaces on these devices are
+  described by their child 'port' nodes. Configuration of a port depends on other
+  devices in the pipeline and is described by 'endpoint' subnodes.
+
+  If a port can be configured to work with more than one remote device on the same
+  bus, an 'endpoint' child node must be provided for each of them. If more than one
+  port is present in a device node or there is more than one endpoint at a port, or
+  port node needs to be associated with a selected hardware interface, a common
+  scheme using '#address-cells', '#size-cells' and 'reg' properties is used.
+
+  All 'port' nodes can be grouped under an optional 'ports' node, which allows to
+  specify #address-cells, #size-cells properties independently for the 'port' and
+  'endpoint' nodes. For example:
+
+    video_device {
+      ...
+      ports {
+          #address-cells = <1>;
+          #size-cells = <0>;
+
+          port@0 {
+              ...
+              endpoint@0 { ... };
+              endpoint@1 { ... };
+          };
+          port@1 { ... };
+      };
+    };
+
+  Two 'endpoint' nodes must be linked with each other via their 'remote-endpoint'
+  phandles. However, Zephyr does not allow circular dependency, so direct phandle
+  references are currently not possible. A 'remote-endpoint-label' string is used
+  instead to be able to specify, at least, the label of the peer remote-endpoint.
+
```

Not sure if I am abusing examples (could hurt readability too), but if interested:

```suggestion    
          sink: endpoint@0 {
              compatible = "zephyr,video-interfaces";
              remote-endpoint = <&source>;
          };

          source: endpoint@0 {
              compatible = "zephyr,video-interfaces";
              remote-endpoint-label = "sink";
          };
```

---

yes, done